knip.config.ts

```diff
@@ -5,7 +5,13 @@ const config: KnipConfig = {
   paths: {
     "@/*": ["./app/*"],
   },
-  entry: ["./app/pages/**/*.{ts,tsx,js,jsx}", "./app/test/colored-console.ts"],
+  entry: [
+    "./app/pages/**/*.{ts,tsx,js,jsx}",
+    "./app/test/colored-console.ts",
+    // TODO Why do I have to put those two entries ?
+    "./app/charts/map/wmts-providers-autocomplete.tsx",
+    "./app/charts/map/wmts-playground.tsx",
+  ],
```

Comment:
@ptbrowne doesn't it restrict knip to only look in files in [entry](https://knip.dev/explanations/entry-files) property now? Maybe you can remove the `entry` altogether to make sure we scan the whole repository? 🤔